fthd_ddr.c

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-#include <linux/random.h>
+#include <linux/prandom.h>
```

Comment:
This would break kernel < v5.9 because `linux/prandom.h` was split from `linux/random.h` then. For kernels < 5.9 should still include `linux/random.h`.

Reply:
Exactly.